src/services/projection/projections/bench.ts

```diff
@@ -152,9 +151,9 @@ export function calculateBenchForSkillForPeriod(
 
   projectionBench = Math.max(...result);
 
-  const employeesBenchValue = arrayOfDays.map((employee) => ({
+  const employeesBenchValue = arrayOfDays.map((employee, i) => ({
     name: employee.employeeName,
-    value: Math.max(...employee.days),
+    value: Math.min(...employee.days),
```

Comment:
Why this change?

Reply:
This may be a mistake, I was testing how different values would appear on the dashboard.

I think the other potential major issue is related to how Confidence is calculated, but that said this probably isn't the solution for that, I'm gonna review the changes for the PR to see if this change is necessary, or if anything else is necessary.

Reply:
undid this change, thanks for the catch.